src/components/Button/index.tsx

```diff
@@ -61,7 +62,7 @@ type ButtonProps = (ButtonWithText | ChildrenProps) & {
     isDisabled?: boolean;
 
     /** A function that is called when the button is clicked on */
-    onPress?: (event?: GestureResponderEvent | KeyboardEvent) => void;
+    onPress?: (event?: GestureResponderEvent | KeyboardEvent, method?: PaymentMethodType) => void;
```

Comment:
```suggestion
    onPress?: (event?: GestureResponderEvent | KeyboardEvent) => void;
```

I don't understand this change

Reply:
its because of that 
<img width="1597" alt="image" src="https://github.com/Expensify/App/assets/25230417/e4a8434c-d95c-4d66-8a33-6def09196e6b">
without this change `Button` component was complaining

Reply:
Hmm doens't make sense to pass such specific param `method?: PaymentMethodType` to a generic component, `Button` won't even use it. We should fix `triggerKYCFlow` to don't send this unused param